packages/router/router.ts

```diff
@@ -954,7 +954,7 @@ export function createRouter(init: RouterInit): Router {
       loadingNavigation = navigation;
 
       // Create a GET request for the loaders
-      request = createRequest(request.url, request.signal);
+      request = new Request(request.url, { signal: request.signal });
```

Comment:
When we already have a valid request, we can just inline the request creation.  `createClientSideRequest` is now for when we _don't_ have a `request` and we just have a `path`